### PR TITLE
OWNERS(releng): Reconcile existing Release Managers

### PR DESCRIPTION
**Pull request by justaugustus**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

Part of kubernetes/sig-release#1713 and kubernetes/sig-release#1789.

- OWNERS(releng): Add Veronica and Nabarun to top-level aliases
- `{build,staging/publishing}/OWNERS`: Reconcile Release Managers
- `test/{images,utils/image}/OWNERS`: Alpha-sort reviewers/approvers
- `{cluster,test}/OWNERS`: Add Release Managers as reviewers for images
- `CHANGELOG/OWNERS`: Reset reviewers/approvers for Kubernetes v1.24

Signed-off-by: Stephen Augustus <foo@auggie.dev>

/assign @dims @liggitt @cpanato @puerco 
/cc kubernetes/release-managers 

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cluster/images/OWNERS
Comment (dims) [inline cluster/images/OWNERS:4]: can you please add `# Release Manager` so we know that this is for their role as a release manager?

unless we can count on folks outside of their release manager role... then we would be looking at how many files/directories they have reviewed in the past etc.

same is true for the other files in this PR as well. actually it may be better to just use the alias from `OWNERS_ALIASES` as well i think.

cc @spiffxp @BenTheElder @cblecker

Comment (justaugustus) [reply cluster/images/OWNERS]: > can you please add `# Release Manager` so we know that this is for their role as a release manager?

Fixed!

> same is true for the other files in this PR as well. actually it may be better to just use the alias from `OWNERS_ALIASES` as well i think.

IIRC, the last time I made changes in some of these directories, reviewers preferred explicitly naming the contributors instead of aliases. I prefer aliases, but I'll wait for a few folks to check in before making additional changes.

Comment (dims) [reply cluster/images/OWNERS]: @justaugustus IMHO, if folks are interested in becoming reviewers, then they should be adding themselves as such. here in this PR, it's a bulk update for a specific reason namely "release managers should be aware of what is happening" more than "i am interested in becoming a reviewer" ..

+1 to wait for others to chime in.

Comment (justaugustus) [reply cluster/images/OWNERS]: > if folks are interested in becoming reviewers, then they should be adding themselves as such

Completely agreed! :)

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Comment by cblecker** [comment]
/hold

Have a couple questions. Will comment this morning.

**Comment by justaugustus** [comment]
> /hold
> 
> Have a couple questions. Will comment this morning.

Any update, @cblecker? Would like to get this updated and unheld soon.

**Comment by cblecker** [comment]
@justaugustus my apologies.. I definitely wrote a comment on this one but looks like it didn't save or I forgot to hit the comment button 🤦 

for places where we are adding folks based on their role, is the access they are getting here explicitly required for the execution of that role?

these are all areas of the project where we need more help, but I'm concerned about the path of folks possibly getting access to these areas because of a role they hold without having the necessary background to understand the implications of changes they might be reviewing/approving.

lots of ways to tackle that though! We could possibly disconnect these areas from a specific role (like release manager) but encourage those folks to go through the regular process to get into owners files (actively reviewing PRs, and then individually requesting elevated access). We could create more docs/training around the types of changes that are okay and how to review/approve changes to these areas. We could scope down the access to only the actual needed access for executing the role of release manager.

To be clear: our release managers are already trusted folks, and I'm not trying to block adding more folks to areas of the project that need it. I just want to make sure that folks in reviewers/approvers lines actually have the needed context and subject matter expertise to be effective in these areas.

Thanks! (and sorry again for the comment mix up)

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Pull request merged**
